magellan/blocker/attr_equiv_blocker.py

```diff
@@ -38,46 +101,108 @@ def block_tables(self, ltable, rtable, l_block_attr, r_block_attr,
         cm._validate_metadata_for_table(ltable, l_key, 'ltable', logger, verbose)
         cm._validate_metadata_for_table(rtable, r_key, 'rtable', logger, verbose)
 
+        # # update the progress bar
+        if show_progress:
+            prog_bar.update() # indicating completion of validation
+   
         # do blocking
 
         # # remove nans: should be modified based on missing data policy
         l_df, r_df = rem_nan(ltable, l_block_attr), rem_nan(rtable, r_block_attr)
```

Comment:
Yes based on the missing value handling policy that is chosen this will get updated. Currently, since the policy is undecided, I went with the policy of the older version.